api/clients/v2/accountant.go

```diff
@@ -109,7 +109,7 @@ func (a *Accountant) BlobPaymentInfo(ctx context.Context, numSymbols uint32, quo
 }
 
 // AccountBlob accountant provides and records payment information
-func (a *Accountant) AccountBlob(ctx context.Context, numSymbols uint32, quorums []uint8, salt uint32) (*core.PaymentMetadata, error) {
+func (a *Accountant) AccountBlob(ctx context.Context, numSymbols uint32, quorums []uint8) (*core.PaymentMetadata, error) {
```

Comment:
why are you removing the salt argument?

Reply:
Since salt is not in payment header anymore we don't need to give it to the accountant

Reply:
Isn't that salt still needed for some things? Trying to understand where this salt is passed/used instead in the new code. Unless the accountant was just never using that salt anyways?

Reply:
The salt was only there so the Accountant can include it in the PaymentMetadata that it later returns. It doesn't do anything special, and probably didn't need to take it in the first place if we added the salt to metadata after Accountant generates it